### DISCUSSION TITLE
Straw removal

### DISCUSSION THREAD

**Discussion by Iris221164** [APSIMInitiative/ApsimX#9437] [Q&A]
how can I remove the straw after harvest?

**Comment by BrianCollinss** [comment]
To be clear, what do you mean by 'remove'? Is it 'just removing from the system', or 'burning', or 'removing from surface and incorporating into soil'?

**Reply by IrisVogeler** [reply]
Hi Brian,

yes removing a certain fraction from the straw or AGB (which is not grain) from the system.  Thanks for your reply

**Comment by IrisVogeler** [comment]
I figured it out myself
